addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -24,6 +24,17 @@
     transform: scale(-1);
 }
 
+// Needed to prevent the emoji picker from inheriting theme colors
+.o-EmojiPicker {
+    --o-EmojiPicker-bg: transparent;
+    --o-EmojiPicker__section-bg: white;
+    --o-EmojiPicker-color: black;
+    --o-EmojiPicker__input-color: #{rgba(black, $o-opacity-disabled)};
```

Comment:
The input  placeholder should be dimmed out, I agree (the value you're probably looking for is `$o-opacity-muted` I guess), but the input value should be a solid black or dark-gray, no? 

| .  | .|
|--------|--------|
| ![image](https://github.com/odoo/odoo/assets/8558351/f759d004-1963-4b9c-b20b-0fe30904ad77) | ![image](https://github.com/odoo/odoo/assets/8558351/ac3c10ef-a6cf-4a82-b378-1b0647957738) |